AddressService/AddressService.Core/Dto/NearestPostcodeDto.cs

```diff
@@ -1,13 +1,16 @@
-﻿using Newtonsoft.Json;
+﻿using System.Runtime.Serialization;
+using Newtonsoft.Json;
 
 namespace AddressService.Core.Dto
 {
     public class NearestPostcodeDto
     {
         [JsonProperty("a")]
+        [DataMember(Name = "a")]
```

Comment:
why the name a?

Reply:
to make it compress and serialise faster (the property name will be shorter) - i've tested that it does make it faster.